api.swagger.yaml

```diff
@@ -1201,7 +2106,7 @@ components:
           type: string
           example: 1f0c6cb5-a3c3-4197-bebc-b3e44462dc23
         timestamp:
-          description: This is timestamp (in seconds) of when the notification was created
+          description: This is timestamp of when the notificaiton was created
```

Comment:
I assume this one was a copy/paste error as it also reintroduces a spelling error ;)

Reply:
updated